v2/cmd/asoctl/cmd/export_template.go

```diff
@@ -24,14 +32,14 @@ type templateOptions struct {
 	crdPattern []string
 }
 
-var expectedVersionRegex = regexp.MustCompile(`^v\d\.\d\.\d$`)
+var expectedVersionRegex = regexp.MustCompile(`^v\d\.\d?\d\.\d$`)
```

Comment:
Ooh good catch

Reply:
Though maybe we just want:
```suggestion
var expectedVersionRegex = regexp.MustCompile(`^v\d\.(\d)+\.(\d)+$`)
```

Reply:
Why do we need the `(` parenthesis `)`? 

I'm thinking we could simplify it to `^v\d\.\d+\.\d+$`.

Reply:
hah, we don't - good point

Reply:
Updated regex.